packages/components/src/Button/Button.tsx

```diff
@@ -188,7 +188,7 @@ function ButtonInternals({ label, icon, size = "base" }: ButtonProps) {
 function getTypeSizes(size: string) {
```

Comment:
Since 2 of the three cases now return "base" do we even need this function anymore? Feels like a check on the Typography component could now suffice.

Reply:
I was thinking something more like changing line 180 to

`size={"large" ? "large" : "base"}`

but that doesn't really change anything and leaving this how it is does give us a better outlet for managing label size going forward, so I think I will in fact leave this alone.

Reply:
Perfect - sounds good to me!